internal/mock/mocks.go

```diff
@@ -101,40 +101,56 @@ func (t Repository) PushSignature(ctx context.Context, mediaType string, blob []
 	return ocispec.Descriptor{}, ocispec.Descriptor{}, nil
 }
 
+type PluginMock struct {
+	Metadata        proto.GetMetadataResponse
+	ExecuteResponse interface{}
+}
+
+func (p *PluginMock) GetMetadata(ctx context.Context, req *proto.GetMetadataRequest) (*proto.GetMetadataResponse, error) {
+	return &p.Metadata, nil
+}
+
+func (p *PluginMock) VerifySignature(ctx context.Context, req *proto.VerifySignatureRequest) (*proto.VerifySignatureResponse, error) {
+	if resp, ok := p.ExecuteResponse.(*proto.VerifySignatureResponse); ok {
+		return resp, nil
+	}
+	return nil, nil
```

Comment:
It doesn't look correct to return nil for both of the outputs. How about:
`return nil, p.ExecuteError`. 
For details: please check https://github.com/patrickzheng200/notation-go/blob/refactor/internal/mock/mocks.go#L118

Reply:
Updated.